test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -361,7 +361,7 @@ export class DisplayTestApp {
     await FrontendDevTools.initialize();
     await HyperModeling.initialize();
     await EditTools.initialize();
-    await MapLayersFormats.initialize();
+    MapLayersFormats.initialize();
```

Comment:
why was this changed?

Reply:
linting error when I ran `rush lint` locally: 
<img width="1387" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/98850418/b2c003bc-a376-4fd5-b333-d3e564806630">

Reply:
let me know if I should revert and push those 2 lint related changes I made

Reply:
the lint error is completely irrelevant to the change you made here

Reply:
i couldve sworn @mdastous-bentley made this async

Reply:
It does look to be async, not sure why got lint error. Let me try it again
<img width="1094" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/98850418/3b735b84-06ba-47f1-8fbe-3d6211983507">